### DISCUSSION TITLE
converting airgeddon evil portal to pineapple format

### DISCUSSION THREAD

**Discussion by merrychrishna** [kbeflo/evilportal-templates#43] [Ideas]
is there a way i or someone else can take the portal from airgeddon and put it into the pineapple?
i like airgeddon because theres no issue with redirecting traffic to the portal like fluxion has an issue with. and i also like how airgeddon gets the handshake and creates a portal from the wifi handshake so you can log in and then youre released back to your original ssid.

ive been looking at this section of the bash script.
https://github.com/v1s1t0r1sh3r3/airgeddon/blob/master/airgeddon.sh#L10709

but im still not sure how to rewrite it into the pineapple. it might be beyond my abilities so my next question is if someone can add it as a feature request for an additional portal that we can place in our pineapples for people who want it.

**Comment by brightplastik** [comment]
I totally second that. All the points illustrated by Chrishna are aces in airgeddon's sleeve, but pineapples are waaaaaay more prectical to use.
